src/PopperContent.js

```diff
@@ -120,28 +112,22 @@ class PopperContent extends React.Component {
       onClosed,
       fade,
       transition,
+      placement,
       ...attrs
     } = this.props;
     const arrowClassName = mapToCssModules(classNames(
       'arrow',
       _arrowClassName
     ), cssModule);
-    const placement = this.state.placement || attrs.placement;
-    const placementFirstPart = placement.split('-')[0];
     const popperClassName = mapToCssModules(classNames(
       _popperClassName,
-      placementPrefix ? `${placementPrefix}-${placementFirstPart}` : placementFirstPart
+      placementPrefix ? `${placementPrefix}-auto` : ''
```

Comment:
This allows the arrow to inherit its style from the computed popper placement instead of the `placement` property passed into this component.

See https://github.com/twbs/bootstrap/blob/c1ee395f80c05de8317588b07f34a65c5b95c42c/scss/_popover.scss#L136